dotnet/aspnetcore/kubernetes/Application/aspnet-core-dotnet-core/aspnet-core-dotnet-core.csproj

```diff
@@ -7,6 +7,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights.Kubernetes" Version="1.0.0-beta2" />
```

Comment:
Do you need this?

Reply:
This dll is used to get kubernetes related properties such as PodName, Deployment etc. It's better to get this info, I suppose.